pages/index.jsx

```diff
@@ -5,19 +5,28 @@ import { PropTypes } from 'prop-types';
 
 import Layout from '../components/Layout';
 import ContentContainer from '../components/content/ContentContainer';
-import { DEFAULT_CONTENT_NODE } from '../constants';
+import settings from '../utilities/siteSettings';
 
-const url = process.env.NODE_ENV !== 'production'
+const episodeLatestUrl = process.env.NODE_ENV !== 'production'
   ? 'http://localhost:3000/api/podcast/latest'
   : 'http://bitfaced.com/api/podcast/latest';
 
+const episodeListUrl = process.env.NODE_ENV !== 'production'
+  ? 'http://localhost:3000/api/podcast/list'
```

Comment:
It would also be nice if at some point we can add the URLs for the `http://localhost:3000` && `http://bitfaced.com` to a constant somewhere for shared usage of `base_url.local` or `base_url.prod`, etc